pkg/install/v1alpha1/localpv_sc.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// TODO
+// Rename this file by removing the version suffix information
+package v1alpha1
+
+const localPVSCYamls = `
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: hostpath
```

Comment:
sc name `hostpath` is generic to k8s, can we name it as  `openebs-hostpath` ? .

Reply:
good point! changed it.